src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -265,14 +265,24 @@
     </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
-    <Reference Include="SonarAnalyzer">
+    <!-- TODO: AMAURY - re-enable when SonarAnalyzer is merged to master and delete hack -->
+    <!--<Reference Include="SonarAnalyzer">
       <HintPath>..\..\binaries\SonarAnalyzer.dll</HintPath>
     </Reference>
     <Reference Include="SonarAnalyzer.CSharp">
       <HintPath>..\..\binaries\SonarAnalyzer.CSharp.dll</HintPath>
     </Reference>
     <Reference Include="SonarAnalyzer.VisualBasic">
       <HintPath>..\..\binaries\SonarAnalyzer.VisualBasic.dll</HintPath>
+    </Reference>-->
+    <Reference Include="SonarAnalyzer">
+      <HintPath>..\..\hack-binaries\SonarAnalyzer.dll</HintPath>
```

Comment:
I cannot add a new feed in nuget as the private builds are not defined as a nuget feed.